### PR TITLE
Backend for superadmin manage page

### LINKED ISSUE #695: Create table for CMS
table with fields according to the role of superadmin. Superadmin should be able to edit: 
- information about municipalities
- add and delete municipalities
- information about companies, for company page 
- add and remove FAQs 
- information on page "Hvordan starte Gi en jul i din kommune"
- add and remove contacts 
- information about contacts 
- information in component "Hvordan fungerer Gi en jul"

### PR DESCRIPTION
**Pull request by En-Lun**
Created backend for superadmin => CMS
created Table

close #695

### CODE REVIEW

File: GiEnJul/Controllers/SuperAdminController.cs [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    [ApiController]
+    public class SuperAdminController : ControllerBase
+    {
+
+        private readonly ICmsRepository _cmsRepository;
+        private readonly IMapper _mapper;
+
+
+        public SuperAdminController(
+
```
Comment (RonHaland) [inline GiEnJul/Controllers/SuperAdminController.cs:23]: -newline

File: GiEnJul/Controllers/SuperAdminController.cs [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+        }
+
+         [HttpGet("getall")]
+         [Authorize(Policy = Policy.SuperAdmin)]
+           public async Task<IEnumerable> GetAllContent([FromQuery] string contentType)
+         {
+            var content = await _cmsRepository.GetCmsByContentType(contentType);
+            return content;
+         }
+        [HttpGet("Getsingle")]
```
Comment (RonHaland) [inline GiEnJul/Controllers/SuperAdminController.cs]: the other requests are all lowercase

File: GiEnJul/Repositories/CmsRepository.cs [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+            var cmsByContentType = GetCmsByContentTypeAsync(contentType);
+
+            return cmsByContentType;
+        }
+
+        public Task<IEnumerable>GetSingleCmsByContentTypeAndIndex(string contentType, string index)
+        {
+            var singleCmsByContentTypeAndIndex = GetSingleCmsByContentTypeAsync(contentType, index);
+            return singleCmsByContentTypeAndIndex;
+        }
```
Comment (RonHaland) [inline GiEnJul/Repositories/CmsRepository.cs:36]: shouldn't return Task if the method is not async.
Are these needed?

**Review by RonHaland — CHANGES_REQUESTED** [review]

File: GiEnJul/Repositories/CmsRepository.cs (hunk 3 above)
Comment (havardMoe) [reply GiEnJul/Repositories/CmsRepository.cs:36]: Agree with Ron!

**Review by MagnusVG — DISMISSED** [review]

File: GiEnJul/Repositories/CmsRepository.cs (hunk 3 above)
Comment (En-Lun) [reply GiEnJul/Repositories/CmsRepository.cs]: fixed

**Review by hildmi — APPROVED** [review]

**Review by RonHaland — APPROVED** [review]

**Pull request merged**